### PR TITLE
Contrast 29288 node2

### PR DESCRIPTION
**Pull request by Avuxo**
Documentation updates for the new agent now that we have C++ dependencies.

### CODE REVIEW

File: content/installation/node/NodeInstall.md [hunk 1]
```diff
@@ -4,6 +4,31 @@ description: "Installing the Node.js Agent"
...
+> **Notes:** 
+ * If you can build other modules with C++ bindings like [node-sass](https://www.npmjs.com/package/node-sass) and [sqlite3](https://github.com/mapbox/node-sqlite3), you probably meet all of the requirements for Contrast.
+ * The official [Node Docker images](https://hub.docker.com/_/node/) come with all of the prerequisites installed.
+
+### Windows
+
+To install the necessary compiler toolchain on Windows, run `npm install -g --production windows-build-tools`. If you're using a Node version older than 8.0, you must use `windows-build-tools` version 3.0 (not the newest version, 4.0).
+
+### macOS
+
```
Comment (kaitlynmortimer) [inline content/installation/node/NodeInstall.md]: @Avuxo It looked like all of these (Windows, macOS, etc.) are supposed be subsections under Prerequisites. If that's wrong, I can readjust the subhead levels.

File: content/installation/node/NodeInstall.md [hunk 2]
```diff
@@ -50,3 +75,7 @@ First, add the following script to your application's *package.json*:
 ```
 
 Then, the agent can be run with ```npm run contrast```. This npm script can be changed to include other runtime configurations, such as an alternate configuration file location. For more information, see [Node Agent Configuration](installation-node.html#node-config).
+
+## Errors during Installation
+
+`Invalid ELF Header` - This error occurs when a native C++ module targeting Linux has been compiled for a non-Linux target. Ensure that you are installing the agent from a compatible system.
```
Comment (kaitlynmortimer) [inline content/installation/node/NodeInstall.md]: @Avuxo Should we add this bit to https://docs.contrastsecurity.com/troubleshooting-node.html#node-ts-install instead?

File: content/installation/node/NodeInstall.md (hunk 1 above)
Comment (Avuxo) [reply content/installation/node/NodeInstall.md]: Yep, that was an oversight on my part.

File: content/installation/node/NodeInstall.md (hunk 2 above)
Comment (Avuxo) [reply content/installation/node/NodeInstall.md]: I'll move it over there; good catch.

**Comment by meandthemoon** [comment]
Would it make sense to document here that the auto-update feature will ignore updates for major version bumps?

**Pull request merged**